source/install/ee-prod-rhel-7.rst

```diff
@@ -190,7 +190,7 @@ Set up Mattermost Server
 
           [Unit]
           Description=Mattermost server
-          After=network.target postgresql-9.4.service
+          After=network.target postgresql-9.4.service mysqld.service
```

Comment:
The unit waits until after postgresql starts, but if the postgresql-9.4.service unit doesn't exist, will Mattermost still get started? Is systemd smart enough to overlook a missing unit?

If not, it might be better to indicate that if using mysql, then replace the postgresql-9.4.service with mysqld.service